tests/e2e/utils/utils.js

```diff
@@ -167,16 +167,18 @@ async function makeRequest(page, endpoint, apiVersion = "v0") {
     const url = host + apiVersion + endpoint;
     console.log("makeRequest", url);
     const resp = await fetch(url);
+    // once the stream is read it cannot be read again
+    const respText = await resp.text();
 
     try {
-      const jsonResp = await resp.json();
+      const jsonResp = await JSON.parse(respText);
```

Comment:
JSON.parse doesn't need an ``await`` as it is synchronous

Reply:
🤦 indeed, thanks